config/fcos/v1_5_exp/translate.go

```diff
@@ -292,3 +302,55 @@ func translateBootDeviceLuks(from BootDeviceLuks, options common.TranslateOption
 	tm.AddTranslation(path.New("yaml"), path.New("json"))
 	return
 }
+
+func (c Config) processPackages(options common.TranslateOptions) (types.Config, report.Report, translate.TranslationSet) {
+	yamlPath := path.New("yaml", "extensions")
+
+	ret := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+
+	if len(c.Extentions) == 0 {
+		return ret, r, ts
+	}
+
+	type TreeFile struct {
+		Packages []string `yaml:"packages"`
+	}
+
+	p := TreeFile{
+		Packages: c.Extentions,
+	}
+
+	marshalledYAML, err := yaml.Marshal(&p)
+
+	if err != nil {
+		r.AddOnError(yamlPath, err)
+		return ret, r, ts
+	}
+
+	treeFilePath := "/etc/rpm-ostree-layer.yaml"
+
+	file := types.File{
+		Node: types.Node{
+			Path: treeFilePath,
+		},
+	}
+
+	src, gzipped, err := baseutil.MakeDataURL([]byte(marshalledYAML), file.Contents.Compression, !options.NoResourceAutoCompression)
```

Comment:
The second argument is always nil, so for clarity I think it's better to pass nil there.

Reply:
If we know that it's always going be nil and gzipped is gonna always be false, any reason to not just ignore `gzipped` and turn it into `_`?

Reply:
`gzipped` won't always be false.  `MakeDataURL` makes a decision about whether to compress.

Reply:
The actual rule is: skip compression if the third argument is `false` or if the second argument is `&"gzip"`.